docs/design/vcpu-handling-runtime-rs.md

```diff
@@ -0,0 +1,51 @@
+# Virtual machine vCPU sizing in Kata Containers 3.0
+
+> Preview: 
+> [Kubernetes(since 1.23)][1] and [Containerd(since 1.6.0-beta4)][2] will help calculate `Sandbox Size` info and pass it to Kata Containers through annotations.
+> In order to adapt to this beneficial change and be compatible with the past, we have implemented the new vCPUs handling way in `runtime-rs`, which is slightly different from the original `runtime-go`'s design.
```

Comment:
nit : could you describe what behavior is different from go runtime? I assume is the support for `Sandbox Size` part ?

Reply:
The way we support sandbox size is different. In runtime-go, only we open the static_sandbox_resource_mgmt, the sandbox size will be effective.